### DISCUSSION TITLE
Communication around babel-plugin-lodash archived?

### DISCUSSION THREAD

**Discussion by istateside** [lodash/lodash#5784] [General]
Hey folks. I noticed that the babel-plugin-lodash repo was archived on September 16th.

Was there a public announcement somewhere about this? I couldn't find anything digging through the issues on that repo.

A deprecation message is shown every time the package is used in a babel config, which is adding a lot of noise to our logs - there's an open issue on the repo about it: https://github.com/lodash/babel-plugin-lodash/issues/259

I _think_ the reasoning is that modern lodash can use tree-shaking to get the same effect as the babel-plugin-lodash, if you're using `lodash-es`. That isn't so straightforward for my team, so at least for the immediate future, we're stuck with babel-plugin-lodash.

I'm not looking for assistance with that deprecation message here (I'll leave that to the linked issue), but was there an announcement made about the repo being archived? Could we maybe update the README for the package to explain why it was archived, and what alternatives there are for consumers? I imagine that might be difficult if the repo was archived, but having no messaging around the archiving was surprising.

**Comment by lkraav** [comment]
Definitely weird how seems complete radio silence around it.
